### ISSUE TITLE
salt-ssh crashes with KeyError: 'state.check_result'

### ISSUE THREAD

**Issue by TheJJ** [saltstack/salt#54945]
### Description of Issue
`salt-ssh` no longer works as it crashes when the minion tries to return the result.
```
KeyError: 'state.check_result'
```

### Steps to Reproduce Issue
```
$ salt-ssh -i --refresh --user $USER 'vmname21' state.sls lsvm test=True -l info
[INFO    ] Loading Saltfile from '/srv/saltstack/Saltfile'
[INFO    ] Caching directory '_modules/' for environment 'base'
[INFO    ] Caching directory '_states/' for environment 'base'
[INFO    ] Caching directory '_grains/' for environment 'base'
[INFO    ] Caching directory '_renderers/' for environment 'base'
[INFO    ] Caching directory '_returners/' for environment 'base'
[INFO    ] Caching directory '_output/' for environment 'base'
[INFO    ] Caching directory '_utils/' for environment 'base'
[ERROR   ] JSON Render failed for: <salt.loader.LazyLoader object at 0x7fe0679150f0>
Traceback (most recent call last):
  File "/var/tmp/.me_3c8639_salt/salt-call", line 27, in <module>
    salt_call()
  File "/var/tmp/.me_3c8639_salt/pyall/salt/scripts.py", line 431, in salt_call
    client.run()
  File "/var/tmp/.me_3c8639_salt/pyall/salt/cli/call.py", line 57, in run
    caller.run()
  File "/var/tmp/.me_3c8639_salt/pyall/salt/cli/caller.py", line 138, in run
    ret = self.call()
  File "/var/tmp/.me_3c8639_salt/pyall/salt/cli/caller.py", line 237, in call
    ret['return'] = self.minion.executors[fname](self.opts, data, func, args, kwargs)
  File "/var/tmp/.me_3c8639_salt/pyall/salt/executors/direct_call.py", line 12, in execute
    return func(*args, **kwargs)
  File "/var/tmp/.me_3c8639_salt/pyall/salt/modules/state.py", line 2199, in pkg
    _set_retcode(ret)
  File "/var/tmp/.me_3c8639_salt/pyall/salt/modules/state.py", line 108, in _set_retcode
    if not __utils__['state.check_result'](ret, highstate=highstate):
  File "/var/tmp/.me_3c8639_salt/pyall/salt/loader.py", line 1208, in __getitem__
    func = super(LazyLoader, self).__getitem__(item)
  File "/var/tmp/.me_3c8639_salt/pyall/salt/utils/lazy.py", line 108, in __getitem__
    raise KeyError(key)
KeyError: 'state.check_result'
[ERROR   ] Expecting value: line 1 column 1 (char 0)
vmcremers21:
    <salt.loader.LazyLoader object at 0x7fe0679150f0>
```

### Versions Report

```
Salt Version:
           Salt: 2019.2.1
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.6.1
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.10
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.5.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: Not Installed
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 3.6.8 (default, Aug 20 2019, 17:12:48)
   python-gnupg: 0.4.1
         PyYAML: 3.12
          PyZMQ: Not Installed
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.5.3
            ZMQ: Not Installed
 
System Versions:
           dist: Ubuntu 18.04 bionic
         locale: UTF-8
        machine: x86_64
        release: 4.15.0-55-generic
         system: Linux
        version: Ubuntu 18.04 bionic 
```

**Comment by TheJJ** [comment]
A downgrade to `2019.2.0` fixes the problem (i just downgraded `salt-ssh` and `salt-common`, nothing else.)

**Comment by TheJJ** [comment]
bad bot

**Comment by Ch3LL** [comment]
and if you try on the 2019.2.2 you see the same thing?

Also can you share your state that is causing this? i cant seem to replicate it.

**Comment by TheJJ** [comment]
The broken pip state bug was fixed in #54826 and #54807
